edb/server/protocol/auth_ext/http.py

```diff
@@ -305,28 +432,51 @@ async def handle_request(
                     if ui_config is None:
                         response.status = http.HTTPStatus.NOT_FOUND
                     else:
-                        providers = util.get_config(
-                            self.db.db_config,
-                            "ext::auth::AuthConfig::providers",
-                            frozenset
-                        )
-                        password_providers = [
-                            p for p in providers
-                            if (util.get_config_typename(p) ==
-                                'ext::auth::PasswordClientConfig')
-                        ]
-                        assert(len(password_providers) == 1)
+                        password_provider = self._get_password_provider()
```

Comment:
Can we return a `404` for these routes if the `password_provider` is `None` instead of throwing an error?